### ISSUE TITLE
Update dependencies version

### ISSUE THREAD

**Issue by RainbowMango** [karmada-io/website#79]
**What would you like to be added**:
Update the dependencies version(in both `package-lock.json` and `yarn.lock`) due to security concerns:
- [x] Upgrade `trim` to version 0.0.3 or later
- [x] Upgrade `browserslist` to version 4.16.5 or later
- [x] Upgrade `dns-packet` to version 1.3.2 or later
- [x] Upgrade `ws` to version 7.4.6 or later
- [x] Upgrade `normalize-url` to version 4.5.1 or later
- [x] Upgrade `tar` to version 6.1.9 or later. 
- [x] Upgrade `path-parse` to version 1.0.7 or later  (#380, @SAMZONG)
- [x] Upgrade `nth-check` to version 2.0.1 or later
- [x] Upgrade `follow-redirects` to version 1.14.8 or later
- [x] Upgrade `node-fetch` to version 2.6.7 or later
- [x] Upgrade `ansi-regex` to version 5.0.1 or later.
- [x] Upgrade `minimist` to version 1.2.6 or later.
- [x] Upgrade `cross-fetch` to version 3.1.5 or later.
- [x] Upgrade `ansi-regex` to version 3.0.1 or later.
- [x] Upgrade `got` to version 11.8.5 or later.

**Why is this needed**:

**Comment by RainbowMango** [comment]
I guess if we upgrade the [docusaurus](https://github.com/karmada-io/website/blob/main/docusaurus.config.js) to a new version might solve all issues list on above.

cc @rgrupesh @Arhell

**Comment by Arhell** [comment]
> > I guess if we upgrade the [docusaurus](https://github.com/karmada-io/website/blob/main/docusaurus.config.js) to a new version might solve all issues list on above.
> > cc @rgrupesh @Arhell
> 
> Which version should we upgrade to?

latest?

**Comment by RainbowMango** [comment]
Yes, we can use the latest [2.0.0-beta.21](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-beta.21).

**Comment by kevin-wangzefeng** [comment]
As https://github.com/facebook/docusaurus/releases/tag/v2.0.1 is now available, we can upgrade to v2.0.1 as it's a more stable version

**Comment by Arhell** [comment]
> As https://github.com/facebook/docusaurus/releases/tag/v2.0.1 is now available, we can upgrade to v2.0.1 as it's a more stable version

i update all to latest version
PR https://github.com/karmada-io/website/pull/141

**Comment by RainbowMango** [comment]
Hi @Arhell , Thanks for doing this. most of the dependencies are done by #156, but still have 4 left.

**Comment by SAMZONG** [comment]
If possible, I would like to participate in the repair work here.

**Comment by RainbowMango** [comment]
Sure, go ahead, appreciate it. 
/assign @SAMZONG

**Comment by SAMZONG** [comment]
#380 upgrade path-parse to 1.0.7

**Comment by RainbowMango** [comment]
Thanks @SAMZONG .  Only two alerts left now.

**Comment by RainbowMango** [comment]
cc @warjiang

**Comment by RainbowMango** [comment]
/close
all the dependencies get updated and don't have any security concerned so far.
